common/models.py

```diff
@@ -34,6 +34,12 @@
     ('blue', 'Blue'),
 ]
 
+COLUMN_CHOICES = [
+    ('6', 'half'),
+    ('4', 'third'),
+    ('3', 'quarter'),
```

Comment:
lets put 1-12 inclusive in this list; maybe theres a layout someone wants two-twelfths, ten-twelfths.